src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -2,224 +2,265 @@ import React from 'react'
-          </div>
-        </div>
-        <div
-          css={`
-            padding: ${compact ? `${3 * GU}px 0 0 ${80 + 3 * GU}px` : '0'};
-          `}
-        >
-          {canUpgrade && onRequestUpgrade && (
-            <Button mode="strong" onClick={onRequestUpgrade}>
-              Upgrade
-            </Button>
-          )}
-        </div>
-      </div>
-      {screenshots.length > 0 && (
-        <Screenshots repo={repo} screenshots={screenshots} />
-      )}
-      <div
-        css={`
-          display: flex;
-          flex-direction: column;
-          justify-content: space-between;
-          width: 100%;
-          flex-direction: ${compact ? 'column' : 'row'};
-        `}
-      >
-        <DetailsGroup compact={compact}>
-          <Heading2>Description</Heading2>
-          <div>{description}</div>
-
-          {!!repoDetails && (
-            <React.Fragment>
-              <Heading2>Details</Heading2>
-              <Markdown text={repoDetails} />
-            </React.Fragment>
-          )}
-        </DetailsGroup>
-        <DetailsGroup compact={compact}>
-          <Heading2>Installed instances</Heading2>
-          {instances.map(({ proxyAddress }) => (
-            <div
-              key={proxyAddress}
-              css={`
-                & + & {
-                  margin-top: ${2 * GU}px;
-                }
-              `}
-            >
-              <LocalIdentityBadge entity={proxyAddress} />
-            </div>
-          ))}
-
-          <Heading2>Source code</Heading2>
-          <div>
-            {sourceUrl ? (
-              <StyledLink href={sourceUrl}>{sourceUrl}</StyledLink>
-            ) : (
-              'No source code link.'
-            )}
-          </div>
+                {screenshots.length > 0 && (
+                  <Screenshots repo={repo} screenshots={screenshots} />
+                )}
+                <div
+                  css={`
+                    display: grid;
+                    grid-template-columns: ${compact ? 'auto' : '60% 40%'};
+                    grid-column-gap: ${compact ? 0 : 5 * GU}px;
+                  `}
+                >
+                  <DetailsGroup>
+                    <Heading2 theme={theme}>Description</Heading2>
+                    <div>{description}</div>
 
-          {!!repoAddress && !!repoAddress && (
-            <React.Fragment>
-              <Heading2>Package Name</Heading2>
-              <StyledLink
-                href={blockExplorerUrl('address', repoAddress, {
-                  networkType: network.type,
-                })}
-              >
-                {repoName}
-              </StyledLink>
-            </React.Fragment>
-          )}
-
-          {repoVersions && (
-            <div
-              css={`
-                margin-top: ${2 * GU}px;
-              `}
+                    {!!repoDetails && (
+                      <React.Fragment>
+                        <Heading2 theme={theme}>Details</Heading2>
```

Comment:
This needs some margin top:

<img width="426" alt="Screen Shot 2019-08-26 at 7 02 07 PM" src="https://user-images.githubusercontent.com/4166642/63708190-05a40280-c834-11e9-9fb5-082db37f4f68.png">